### PR TITLE
OCPQE-166730 Define builds for new images and onboard in prow

### PR DESCRIPTION
**Pull request by sgoveas**

### CODE REVIEW

**Review by aleskandro — APPROVED** [review]
Thanks @sgoveas 

/lgtm
/approve

**Comment by aleskandro** [comment]
@sgoveas all right, the dnsmasq image needs a fix (you were installing with apk on top of cs9).

/pj-rehearse skip

File: ci-operator/config/openshift-qe/baremetal-qe-infra/openshift-qe-baremetal-qe-infra-master.yaml [hunk 1]
```diff
@@ -32,6 +32,18 @@ images:
   dockerfile_path: Containerfile
   from: fcos-base
   to: fcos-bastion
+- context_dir: images/bind9
+  dockerfile_path: Dockerfile
+  from: fcos-bastion
```
Comment (aleskandro) [inline ci-operator/config/openshift-qe/baremetal-qe-infra/openshift-qe-baremetal-qe-infra-master.yaml:37]: this should change to `centos8`

File: ci-operator/config/openshift-qe/baremetal-qe-infra/openshift-qe-baremetal-qe-infra-master.yaml [hunk 2]
```diff
@@ -32,6 +32,18 @@ images:
...
+  to: bmqe-bind9
+- context_dir: images/tang
+  dockerfile_path: Containerfile
+  from: fcos-bastion
+  to: bmqe-tang
+- context_dir: images/dhcp
+  dockerfile_path: Containerfile
+  from: fcos-bastion
```
Comment (aleskandro) [inline ci-operator/config/openshift-qe/baremetal-qe-infra/openshift-qe-baremetal-qe-infra-master.yaml:45]: this should change to `centos9` (depending on openshift-qe/baremetal-qe-infra#24)

File: ci-operator/config/openshift-qe/baremetal-qe-infra/openshift-qe-baremetal-qe-infra-master.yaml [hunk 3]
```diff
@@ -32,6 +32,18 @@ images:
   dockerfile_path: Containerfile
```
Comment (aleskandro) [inline ci-operator/config/openshift-qe/baremetal-qe-infra/openshift-qe-baremetal-qe-infra-master.yaml:32]: lines 2 and l68, change to centos8.

after line5, add

```
  centos9:
    name: centos
    namespace: openshift
    tag: stream9
```

File: ci-operator/config/openshift-qe/baremetal-qe-infra/openshift-qe-baremetal-qe-infra-master.yaml [hunk 4]
```diff
@@ -32,6 +32,18 @@ images:
...
   from: fcos-base
   to: fcos-bastion
+- context_dir: images/bind9
+  dockerfile_path: Dockerfile
+  from: fcos-bastion
+  to: bmqe-bind9
+- context_dir: images/tang
+  dockerfile_path: Containerfile
+  from: fcos-bastion
```
Comment (aleskandro) [inline ci-operator/config/openshift-qe/baremetal-qe-infra/openshift-qe-baremetal-qe-infra-master.yaml:41]: change to `centos8`.

**Comment by sgoveas** [comment]
Fixes in https://github.com/openshift/release/pull/44897

**Pull request merged**